screens/OrdersScreen.js

```diff
@@ -72,28 +82,73 @@ class OrdersScreen extends React.Component {
         })
       }
       bottomDivider
-      containerStyle={[ styles.dynamicVerticalPadding(12), { padding: 0 }]}
-     />
+      containerStyle={[styles.dynamicVerticalPadding(12), { padding: 0 }]}
+    />
   )
 
   //https://stackoverflow.com/questions/48061234/how-to-keep-scroll-position-using-flatlist-when-navigating-back-in-react-native
   handleScroll = event => {
     this.setState({ scrollPosition: event.nativeEvent.contentOffset.y })
   }
 
+  handleOrderFilter = values => {
+    const fromDate = values.fromDate
+    const toDate = values.toDate
+    const dateRange = values.dateRange
+
+    if (fromDate && toDate && !dateRange) {
```

Comment:
As mentioned in previous comment, you can simplify this section by passing all three parameters (dateRange, fromDate, toDate) to the backend. The backend will know how to interpret the parameters and return the correct result.

Reply:
passing 3 parameters get java err.